posts/2017-04-17-get-your-lambdas-warm.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Get your Lambdas Warm
+description: How to make your cold lambdas hot with WarmUP plugin
+date: 2017-04-17
+layout: Post
+authors:
+  - GoncaloNeves
+---
+
+# Let's make your lambdas warm
+
+*Cold start in computing = duration of time that takes to boot a system*
+
+In this blog post we tackle this issue with AWS Lambda + Serverless.
+
+## Cold start in Serverless ❄️ *- problem*
+
+Now more than ever with *Function as a Service (FaaS)* cold start is an issue that is detrimental to this huge step forward in the developer world.
+
```

Comment:
Would propose to add a not too technical quick explanation at the top here. Basically a "explained to a 5yo" section. After that the more technical details could be discussed (as it's done below).